### ISSUE TITLE
iOS clearDrawing changes page layout

### ISSUE THREAD

**Issue by huehnerhose** [bradmartin/nativescript-drawingpad#8]
Hi,

I want to show some information "behind" the drawing pad. That way the signer knows what he signs. My page.xml looks like:

```
<Page
    xmlns="http://schemas.nativescript.org/tns.xsd"
    xmlns:DrawingPad="nativescript-drawingpad"
    loaded="pageLoaded"
    >

    <GridLayout rows="*, auto">
        <AbsoluteLayout
            row="0"
            id="outerContainer"
            horizontalAlignment="center"
            >


            <StackLayout
                horizontalAlignment="center"
                verticalAlignment="center"
                id="backgroundLabelContainer"
                >

                <Label text="{{ jobsite }}" />
                <Label text="{{ date }} " />
                <Label text="{{ worktime | durationRowConverter }}" />
                <Label text="{{ breaktimes | breaktimesConverter }}" textWrap="true" />
            </StackLayout>

            <DrawingPad:DrawingPad
                id="drawingPad"
                penColor="#000F55"
                />


        </AbsoluteLayout>

        <Button row="1" text="Unterschrift löschen" tap="resetSignature"/>

    </GridLayout>

</Page>
```

(The GridLayout is there for FloatingButton for Android)

That show the Label in StackLayout "behind" my DrawingPad. It works under iOS and Android. But on `resetSignature`, which calls `clearDrawing` on DrawingPad, Android simply resets the DrawingPad, iOS "hides" the Labels. 
I tried to find out what clearDrawing does, but ended in `_ios.clear()` which I didn't find anywhere. 

Is there a better way to show somehting "behind" or is there a way to not hide my labels?

thanks!

**Comment by isaacfi** [comment]
I have the same issue, the clear removes all in the stacklayout

**Comment by JillevdW** [comment]
After some digging I've found that this behaviour isn't caused by the plugin, it's caused by the underlying CocoaPod. This all happens because there's a `UILongPressGestureRecognizer` registered that isn't exposed to us. I'll be trying to fix this and will submit a PR if I succeed.

**Comment by JillevdW** [comment]
#52

**Comment by bradmartin** [comment]
Closed via #52
